docs/Training-PPO.md

```diff
@@ -73,10 +56,23 @@ Typical Range: `32` - `2048`
 
 #### Max Steps
 
-`max_steps` corresponds to how many steps of the simulation (multiplied by frame-skip) are run durring the training process. This value should be increased for more complex problems.
+`max_steps` corresponds to how many steps of the simulation (multiplied by frame-skip) are run during the training process. This value should be increased for more complex problems.
 
 Typical Range: `5e5 - 1e7`
 
+#### Beta (Used only in Discrete Control)
```

Comment:
Newer research (see https://arxiv.org/abs/1801.01290 for example) shows that entropy bonuses are good for continuous control too, so it now applies to both actually.